src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -51,127 +51,134 @@ public void setup() {
     public void getAccount_whenExist_returnOneAccount() {
         // given
         final var fakeAccountId = fakeDataGenerator.generateRandomInteger();
-        final var fakeActive = fakeDataGenerator.generateRandomBoolean();
-        final var fakeDateCreated = fakeDataGenerator.generateRandomDate();
-        final var fakePetList = fakeDataGenerator.generateDefaultPetList();
-        final var fakePeopleList = fakeDataGenerator.generatePersonList(fakeDataGenerator.generateRandomInteger());
-        final var fakeAccount = fakeDataGenerator.generateAccount(fakeAccountId, fakeActive, fakeDateCreated, fakePetList, fakePeopleList);
+        final var fakeAccount = fakeDataGenerator.generateDefaultAccount();
+        final var fakeAccountResponse = fakeDataGenerator.generateDefaultAccountResponse();
         given(mockAccountRepository.findById(fakeAccountId)).willReturn(Optional.of(fakeAccount));
+        given(mockMapper.map(Optional.of(fakeAccount), AccountResponse.class)).willReturn(fakeAccountResponse);
 
         // when
         final var actual = serviceUnderTest.getAccount(fakeAccountId);
 
         // then
-        then(actual).isEqualTo(fakeAccount);
+        then(actual).isEqualTo(fakeAccountResponse);
     }
 
     @Test
-    public void addAccount_whenValidInput_returnsValidInput() {
+    public void getAccount_whenNotExist_returnEmptyAccountResponse() {
         // given
         final var fakeAccountId = fakeDataGenerator.generateRandomInteger();
-        final var fakeActive = fakeDataGenerator.generateRandomBoolean();
-        final var fakeDateCreated = fakeDataGenerator.generateRandomDate();
-        final var fakePetList = fakeDataGenerator.generateDefaultPetList();
-        final var fakePeopleList = fakeDataGenerator.generatePersonList(fakeDataGenerator.generateRandomInteger());
-        final var fakeAccount = fakeDataGenerator.generateAccount(fakeAccountId, fakeActive, fakeDateCreated, fakePetList, fakePeopleList);
+        final var fakeAccountResponse = new AccountResponse();
```

Comment:
can we rename this variable to emptyAccountResponse

Reply:
renamed emptyAccountResponse